### ISSUE TITLE
Consider a different colour scheme for the output

### ISSUE THREAD

**Issue by kblin** [gamcil/synthaser#9]
Hi folks,

The current output visualisation uses a colour scheme that assigns very similar colours to domains that tend to occur adjacent to each other. This makes it a bit tricky to see what's going on when you have regular colour vision, and pretty much impossible if your colour vision is deficient.
Ideally, you'd want to ensure that adjacent domains have colours with distinct saturations.

**Comment by gamcil** [comment]
Yeah this is an issue I have with clinker too (which has the same colour implementation). The colours are automatically taken at equal increments from a continuous rainbow scale based on the number of domains that have been picked up. It's okay when there aren't so many domains in the visualisation since they're spaced further apart in the scale, but as soon as that number grows there is less and less distinction. I've tried looking into categorical and colourblindness-friendly schemes before (e.g. the Okabe one https://mikemol.github.io/technique/colorblind/2018/02/11/color-safe-palette.html), but found they're usually limited to a relatively small number of colours, which in some searches won't cover all domains without repeats (hence the automatic rainbow way...). Have you got any suggestions? In antiSMASH, you have specific colours for certain domains right? Maybe a quick fix would be to alternate which side of the rainbow scale colours are taken from, which should increase the contrast a little at least.

**Comment by kblin** [comment]
For the domains we use specific colours, but if you want to stick with a rainbow palette you can have a look at our ClusterBlast drawing logic, where the code does try to assign very different colours to adjacent genes in the cluster results.
